src/System.Data.Common/System.Data.Common.sln

```diff
@@ -26,8 +26,8 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{B473F77D-4168-4123-932A-E88020B768FA}.Debug|Any CPU.ActiveCfg = netstandard-Debug|Any CPU
-		{B473F77D-4168-4123-932A-E88020B768FA}.Debug|Any CPU.Build.0 = netstandard-Debug|Any CPU
+		{B473F77D-4168-4123-932A-E88020B768FA}.Debug|Any CPU.ActiveCfg = netcoreapp-Debug|Any CPU
```

Comment:
Shouldn't netcoreapp-Debug|Any CPU be added in addition to `netstandard-Debug|Any CPU`?
Was this modified manually or did VS overwrite the existing files?